recipes-ros/rosbridge-suite/rosbridge-library_0.7.16.bb

```diff
@@ -3,6 +3,6 @@ SECTION = "devel"
 LICENSE = "BSD"
 LIC_FILES_CHKSUM = "file://package.xml;beginline=10;endline=10;md5=68b329da9893e34099c7d8ad5cb9c940"
 
-DEPENDS = "message-generation std-msgs geometry-msgs rospy python-six python-bson python-pytz"
+DEPENDS = "message-generation std-msgs geometry-msgs rospy python-six python-pymongo python-pytz python-html python-imaging python-twisted"
```

Comment:
Commit is subsumed by other changes.

This is a valuable commit; this issue is however already addressed in
bmwcarit#509.